content/docs/composition-vs-inheritance.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: composition-vs-inheritance
-title: Composition vs Inheritance
+titlu: Composition vs Inheritance
```

Comment:
From my understanding of markdown files attributes like: title, prev, next should be left unchanged (so no translation to titlu, anterior, urmator) because ReactJS is made with Gatsby which uses those attributes to set the title of the page and link to prev/next articles. So you should translate only Composition vs Inheritance.